### PR TITLE
10k/50k Word Lists for Dictionaries

### PR DESCRIPTION
**Pull request by Lyxal**
5k just wasn't long enough for the short, and 50k was the best for long with 10k short.

### CODE REVIEW

File: dict_scripts/sss_compress.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                    dp[i] = min(
+                        dp[i],
+                        dictionary(dp[i + j], str[i : i + j], i != 0),
+                    )
+                except ValueError:
+                    pass
+
+        return min(
+            [
+                "“{0}”".format(str),
```
Comment (mathcat4) [inline dict_scripts/sss_compress.py:74]:
```suggestion
        return '"{0}”'.format("".join(compressed[::-1]))

    def optimal(str):
        dp = [0] * -~len(str)
        for i in range(len(str) - 1, -1, -1):
            dp[i] = character(dp[i + 1], str[i])
            for j in range(1, min(len(dp) - i, end_len)):
                try:
                    dp[i] = min(
                        dp[i],
                        dictionary(dp[i + j], str[i : i + j], i != 0),
                    )
                except ValueError:
                    pass

        return min(
            [
                '"{0}"'.format(str),
```

wrong apostrophes?

**Review by mathcat4 — CHANGES_REQUESTED** [review]

File: dict_scripts/sss_compress.py (hunk 1 above)
Comment (Steffan153) [reply dict_scripts/sss_compress.py:74]: No, this is Jelly's string delimiters.

File: dict_scripts/sss_compress.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        compressed = []
+        while z:
+            z, c = divmod(
+                z - 1, 252
+            )  # Taken inspiration from Jelly's bijective base conversion algorithm
+            compressed.append(code_page[c])
+        return "“{0}»".format("".join(compressed[::-1]))
+
+    def optimal(str):
+        dp = [0] * -~len(str)
```
Comment (Steffan153) [inline dict_scripts/sss_compress.py:60]: Come on, this isn't code golf

Comment (Steffan153) [reply dict_scripts/sss_compress.py:60]:
```suggestion
        dp = [0] * (len(str) + 1)
```

File: dict_scripts/sss_compress.py (hunk 1 above)
Comment (Lyxal) [reply dict_scripts/sss_compress.py:74]: This is only to get length under compression. The strings produced are irrelevant.

File: dict_scripts/sss_compress.py (hunk 2 above)
Comment (Lyxal) [reply dict_scripts/sss_compress.py:60]: Don't blame me - blame the script I found on a code golf answer.

**Review by mathcat4 — APPROVED** [review]
lgtm

File: dict_scripts/sss_compress.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                        dictionary(dp[i + j], str[i : i + j], i != 0),
+                    )
+                except ValueError:
+                    pass
+
+        return min(
+            [
+                "“{0}”".format(str),
+                go(dp[0]),
+            ],
```
Comment (Steffan153) [inline dict_scripts/sss_compress.py:76]: You don't need the square brackets here

**Pull request merged**